FESS-GUI/graph.cpp

```diff
@@ -154,7 +154,7 @@ void LocationGraph::setupPlot(QCustomPlot *plot, bool isMain, std::vector<QColor
 
 void LocationGraph::addData(std::vector<QPointF> points)  //function to add xy data to graphs
 {
-    for(int i = 0; i < points.size() && i < numPoints; i++){
+    for(unsigned int i = 0; i < points.size() && i < (unsigned int)numPoints; i++){
```

Comment:
size() returns an unsigned int so I was getting this annoying comparison warning when I compiled the program.. I guess if no one else has that issue we can revert this.